### ISSUE TITLE
apollo:client download-schema conflicts with graphql-code-generator

### ISSUE THREAD

**Issue by baeharam** [apollographql/apollo-tooling#2229]
**Context**

I'm downloading schema using `apollo:client download-schema` and generating types using `graphql-code-generator`. I have files named `queries.ts` for each component as fragment. When I generates types, it has also same queries within types file.

<Example>

`ComponentA/queries.ts`

```gql
query ExampleQuery {
  ...some query
}
```

`src/generated/graphql.tsx` (generated by graphql-code-generator)

```gql
...some types
query ExampleQuery {
  ...some query 
}
...some types
```
When I try to download new schema from our server, it shows error
```bash
yarn apollo:client download-schema ....
```
>CLIError: Error in "Loading queries for Unnamed Project": Error: ️️There are multiple definitions for the `ExampleQuery` operation. Please rename or remove all operations with the duplicated name before continuing.

**TL;DR**

make query --> generate types(generate same queries to generate hooks) --> two same queries --> download schema --> CONFLICT.. why???

**Intended outcome:**

No error! because it only download schema from server. Why it is related with `query`?

**Actual outcome:**

Above error
